pkg/harness/controller/controller.go

```diff
@@ -128,6 +137,14 @@ func (in *stackRunController) toExecutable(ctx context.Context, step *gqlclient.
 	)
 }
 
+func (in *stackRunController) workdir() string {
```

Comment:
Working dir is a harness app argument. It is used in a couple more places, i.e. to prepare the environment, download and unpack tarball, read/save terraform related files such as plan/state. It would be best if job controller could read and configure it during job creation.

Reply:
The idea behind this workdir is it only controls the directory under which commands are executed.  The case is:

* sync in the entire terraform folder (with adjacent modules)
* execute under some subfolder

Reply:
Done. Tested and seems to be working fine.